Dockerfile

```diff
@@ -15,4 +15,11 @@ ENTRYPOINT poetry run flask run --host=0.0.0.0
 FROM base as development
 WORKDIR /opt/todo_app
 ENV FLASK_DEBUG="true"
-ENTRYPOINT poetry run flask run --host=0.0.0.0
\ No newline at end of file
+ENTRYPOINT poetry run flask run --host=0.0.0.0
+
+FROM base as test
+WORKDIR /opt/todo_app
+ENV PYTHONPATH="/opt/todo_app"

```

Comment:
This solves the problem similarly, but I'll just flag an alternative, which is adding an empty `__init__.py` file to the `tests` folder - which is the approach we've taken in the existing `todo_app` folder. These were historically meant to identify Python "modules" (directories) and while my reading is that they aren't meant to be _necessary_ any more, there are still environments and libraries where that's not yet true, so they're a good habit to have (although I can't verbalise why I prefer them to modifying the PYTHONPATH, so feel free to ignore!)